src/modules/project/components/ProjectLists/ProjectLists.tsx

```diff
@@ -1,7 +1,146 @@
+import React, { useEffect } from "react";
+import { useState } from "react";
+import { axiosInstance,  PROJECT_URL } from "../../../../services/EndPoints";
+import { Button, Dropdown, Modal } from "react-bootstrap";
+import { toast } from "react-toastify";
+
+
+interface projectData {
+  title:string;
+  description: string;
+  task:object[];
+  id:number;
+} 
+
 export default function ProjectLists() {
+  const [projectsList, setProjectsList] = useState([]);
```

Comment:
I suggest using search params for filtration and pagination. This will enable preserving the state when reloading or sharing links. Currently, if we reload the page, all filteration and pagination will be reset. React router package provides us with useSearchParams hook.